paddle/fluid/operators/fill_constant_op.h

```diff
@@ -26,7 +26,8 @@ namespace operators {
 
 using Tensor = framework::Tensor;
 
-inline framework::DDim GetShape(const framework::ExecutionContext &ctx) {
+inline framework::DDim GetShape(const framework::ExecutionContext &ctx,
+                                std::string op_type) {
```

Comment:
A suggestion： use `GetDataFromTensor` and `GetDataFromTensorList` to compatible with `int32` and `int64`

Reply:
Done